src/cljs/rems/navbar.cljs

```diff
@@ -60,6 +60,10 @@
               text (get-in page [:translations language :title] (text :t/missing))]
           [nav-link url text])))))
 
+(defn logo [menu-navigation?]
+  [:div {:class (str "navbar-brand" (if menu-navigation? " logo-menu" " logo"))}
```

Comment:
I think this is probably wrong. For example now you are putting the class `navbar-brand` also on the hero logo.

Reply:
ok, I do not get how this is wrong

Reply:
The logo in the main content area is not in the navbar, so it should not have `navbar-brand` class which is for the logo in the navbar?

Reply:
hah, good point